Library/Homebrew/cask/cask.rb

```diff
@@ -169,6 +169,8 @@ def eql?(other)
     def to_h
       {
         "token"          => token,
+        "full_token"     => "#{tap.name}/#{token}",
```

Comment:
```suggestion
        "full_token"     => full_name,
```

We have a `full_name` method

Reply:
I don't know if `full_name` or `full_token` is better to use here. Using token seems more consistent with the token/name difference between formulae and casks, but the method is called `full_name`.

Maybe someone from @Homebrew/cask can weigh in on what's best for consistency here.

Reply:
OK! I'll replace it by `full_name`. I don't mind change it later to `full_token` if it is better that way.

Reply:
One question here.

`full_name` is always `tap.name/token`? Or it is replaced by `token` when `tap.name` value is `homebrew/cask`?

I have seen that `full_name` for formulae is the same as `name` when the tap is `homebrew/core`

Reply:
Done!

Reply:
> One question here.
> 
> `full_name` is always `tap.name/token`? Or it is replaced by `token` when `tap.name` value is `homebrew/cask`?
> 
> I have seen that `full_name` for formulae is the same as `name` when the tap is `homebrew/core`

I've tested this change, and `full_name` is the same as `token` when taps are `homebrew/cask` and `homebrew/cask-fonts`

Reply:
Tests `test/cask/cmd/list_spec.rb` have failed because of this issue.

Should I change the tests to take into account this behavior? Or do we always want to retrieve the tap name inside the `full_name`?

Reply:
That's correct. `full_name` is identical to `name`/`token` for official taps (meaning taps that are `homebrew/*`). This is true for both formulae and casks.

> Should I change the tests to take into account this behavior?

Yes, please!

Reply:
Done!